src/styles/blocks/comparing.scss

```diff
@@ -0,0 +1,64 @@
+.comparing {
+  &__photo {
+    width: 100%;
+
+    transition: 1s;
+    margin-bottom: 32px;
+  }
+
+  &__text {
+    transition: 1.4s;
+  }
+
+  &__link {
+    grid-column: 1 / 5;
+  }
+
+  &__title {
+    font-weight: 700;
+    font-size: 24px;
+    line-height: 140%;
+    margin-bottom: 8px;
+  }
+
+  &__description {
+    font-weight: 400;
+    font-size: 16px;
+    line-height: 140%;
+    margin-bottom: 16px;
+  }
+
+  &__price {
+    font-weight: 700;
+    font-size: 24px;
+    line-height: 140%;
+
+    margin-bottom: 40px;
+
+    &:last-child {
+      margin-bottom: 0;
+    }
+  }
+
+  &__bike {
+    &:hover {
```

Comment:
if I understood correct, I can't do like this without &__bike because it's my card and add animations to the card, cause comparing it's my hole section and add this hover style for all components